workspace/src/app/views/shared/RuleEditor/view/ruleNode/RuleNodeMenu.tsx

```diff
@@ -103,6 +111,13 @@ export const RuleNodeMenu = ({
                         )}
                     />
                 ) : null}
+                <MenuItem
```

Comment:
Please update the node menu accordion our latest spec, moving the "remove" menu item to the bottom and separate it by a divider.